galasa-ui/src/app/layout.tsx

```diff
@@ -22,6 +24,7 @@ export default function RootLayout({ children }: { children: React.ReactNode })
       <body>
         <PageHeader galasaServiceName={galasaServiceName} />
         {children}
+        <Footer serviceHealthyPromise={getServiceHealthStatus()} clientVersionPromise={getClientApiVersion()}/>
```

Comment:
This is good - just need to move away from using server actions for getting data (see https://github.com/galasa-dev/webui/pull/103#discussion_r2060557858)